### ISSUE TITLE
Change XGB learning rate default value

### ISSUE THREAD

**Issue by csala** [HDI-Project/MLPrimitives#21]
XGBClassifier and XGBRegressor `learning_rate` default value is wrongly set to 0.

It should be changed to 0.1, which is the default value of the original primitives.
